README.md

```diff
@@ -1 +1,226 @@
 # origin-metrics
+
+TODO: write brief introduction here.
+
```

Comment:
Consider adding something here about what role you need to run the commands in the following instructions.